kani-driver/src/args.rs

```diff
@@ -43,18 +43,17 @@ pub struct KaniArgs {
     /// Generate visualizer report to <target-dir>/report/html/index.html
     #[structopt(long)]
     pub visualize: bool,
-    /// Generate executable trace test case and print it to stdout.
+    /// Generate concrete playback unit test and either 1) just print it to stdout or 2) add it in-place to the source code.
     /// This option does not work with `--output-format old`.
     #[structopt(
```

Comment:
I don't know how to use kebab-case for the arguments.

Reply:
I considered allowing users to use `--concrete-value` without an argument (and default to `print`). I spent about 30 mins looking through this, and I couldn't find any clean way to do it. Here's what I found:
1. I could do it if I made `concrete_playback: Option<Option<ConcretePlaybackMode>>`. However, that has a really bad effect on code quality IMO. No concrete playback would be `None`, concrete playback in print mode would be `Some(None)`, and concrete playback in in-place mode would be `Some(Some(InPlace))`.
2. Even if I set the default value for `ConcretePlaybackMode` to be `Print`, I'm not allowed to add a `default_value` option to `structopt` because the type is an `Option`.
3. [structopt #123](https://github.com/TeXitoi/structopt/issues/123) is about the exact same thing. They closed it with a PR, but I can't seem to figure out what the PR actually does.

Because I didn't find any clean way to implement this, I decided to force the user to specify either `Print` or `InPlace`. This isn't ideal, but it does have the plus-side of making the operating mode explicit.